cmd/entrypoint/notify.go

```diff
@@ -109,7 +111,7 @@ func notifyWebhookUrl(ctx context.Context, name, xurl string) bool {
 		if errors.Is(err, syscall.ECONNREFUSED) {
 			// We couldn't succesfully connect to the sidecar, probably because it hasn't
 			// started up yet, so we log the error and return false to signal retry.
-			log.Println(err)
+			dlog.Errorf(context.TODO(), err)
```

Comment:
what's wrong with the context passed to this function?

Reply:
good call